tox.ini

```diff
@@ -1,125 +1,35 @@
 [tox]
-minversion = 3.5.0
-skipsdist = True
-skip_missing_interpreters = True
+minversion = 3.18.0
 isolated_build = True
-toxworkdir = {env:TOXWORKDIR:/tmp/{env:USER}/tox}
-envlist = lint
-          py3-agent
-          datalog
-          postprocess
-          tool-scripts
-          util-scripts
-          bench-scripts
-          py3-server
-          server
+toxworkdir = {env:TOXWORKDIR:/var/tmp/{env:USER}/tox}
+distshare = {toxworkdir}/distshare
+envlist = py
 
 [testenv]
-usedevelop = True
+description = Lints code and runs all agent and server unit/functional tests
 install_command = pip install --no-cache-dir --progress-bar off --prefix={envdir} {opts} {packages}
 passenv =
     PY_COLORS
     NO_COLORS
     TERM
     USER
+    PBENCH_UNITTEST_SERVER_MODE
+    PYTHONPATH
+    COV_REPORT_XML
```

Comment:
We need to add `HOME` to this list, to enable Python in the test environment to find the `--user` installation without using an inherited `PYTHONPATH`.

Also, I think we should remove `PYTHONPATH` from this list:  I don't think we want to have it inherited explicitly, since the test environment is supposed to be self-contained and we don't want it polluted by an outside definition.  Note that, if `PYTHONPATH` is explicitly defined in the user's environment, [Tox will pass it in _anyway_](https://tox.wiki/en/latest/config.html#conf-passenv), but if we don't have it listed here, then Tox should issue a warning, which I think is a good idea.